src/Domain/Entity/Broadcast.php

```diff
@@ -125,4 +137,15 @@ public function isRepeat(): bool
     {
         return $this->isRepeat;
     }
+
+    /**
+     * Returns true if the broadcast is on air at a given date.
+     * Broadcast starts are inclusive and ends are exclusive.
+     * Given two broadcasts, the first will end at at 06:30:00 and the second
+     * will start at 06:30:00.
+     */
+    public function isOnAir(DateTimeImmutable $date): bool
```

Comment:
I know it's fussy as it's only a variable name used internally, but this isn't a date. It's a date/time. So should the variable actually be `$datetime`?  
If this was in a single project, I wouldn't be this fussy, but it's in a composer package that may be used by a number of people and so correctness/readabilty seems important.